packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -40,9 +40,12 @@ class Icon extends React.Component {
   }
 
   render() {
-    const filteredProps = _omit(this.props, ['dense', 'onLoaded', 'position', 'theme'])
```

Comment:
Why was filteredProps removed? It ensures that only props are passed down to react-fontawesome which are consumed. All props which are only consumed by Icon are removed by _omit.

Reply:
Yes, i know. So what is there in "props" except the icon attribute which i handle separately?